core/src/main/java/org/jboss/set/mjolnir/archive/batch/RemoveArchieveUserBatchlet.java

```diff
@@ -0,0 +1,96 @@
+package org.jboss.set.mjolnir.archive.batch;
+
+import org.jboss.logging.Logger;
+import org.jboss.set.mjolnir.archive.GitDeleteRepository;
+import org.jboss.set.mjolnir.archive.configuration.Configuration;
+import org.jboss.set.mjolnir.archive.domain.RepositoryFork;
+
+import javax.batch.api.AbstractBatchlet;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityTransaction;
+import java.sql.Timestamp;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Named
+public class RemoveArchieveUserBatchlet extends AbstractBatchlet {
+
+    private final Logger logger = Logger.getLogger(getClass());
+
+    @Inject
+    private EntityManager em;
+
+    @Inject
+    private Configuration configuration;
+
+    @Override
+    public String process() throws Exception {
+
+        logger.infof("User removal process started");
+        boolean successful = false;
+        if (configuration.getRemoveArchives() == true) {
+            List<RepositoryFork> removals = loadRepositoryForks();
+            List<RepositoryFork> filteredList = getFilterDeleteList(removals);
+            for (RepositoryFork repositoryFork : filteredList) {
+                GitDeleteRepository gitDeleteRepository = new GitDeleteRepository(configuration.getRepositoryArchiveRoot() + "/" + repositoryFork.getSourceRepositoryName(),
+                        repositoryFork.getRepositoryName().substring(0, repositoryFork.getRepositoryName().indexOf('/')));
+
+                gitDeleteRepository.gitRemoveBranches();
+                successful = true;
+            }
+
+            if (successful) {
+                return "DONE";
+            } else {
+                return "DONE_WITH_ERRORS";
+            }
+        } else {
+            return "APPLICATION_REMOVE_ARCHIEVE_NOT_APPLICABLE";
```

Comment:
Can we turn all these return strings into constants? E.g. make a `Constants` class in the `batch` package and define all the constants there. You can replace ale the strings in the other batchlet, to keep it tidy.

Reply:
Added these changes